### DISCUSSION TITLE
Imagemagick fails to convert a .dng to jpeg. Any help is aprreciated

### DISCUSSION THREAD

**Discussion by Govind-EIX** [ImageMagick/ImageMagick#2203] [Help]
we are trying to convert a .dng file into jpeg file using imagemagick ,But imagemagick throws an error. Please find the command below

C:\> magick.exe dng:D:\New_folder\DNGTEST_SAMPLE.dng -intent relative -sample 1024x768> -quality 95 -colorspace sRGB -strip -auto-orient -density 72 D:\New_folder\DNGTEST_SAMPLE-S.JPG

Imagemagick version we are using:ImageMagick 7.0.8-3 Q16 x64 2018-06-24

Result:
magick.exe: Unsupported file format or not RAW file `D:\New_folder\DNGTEST_SAMPLE.dng' @ error/dng.c/ReadDNGImage/305.

**Comment by fmw42** [comment]
Can you post a link to your DNG file?

**Comment by Govind-EIX** [comment]
Attached the file inside the zip folder.
[Test.zip](https://github.com/ImageMagick/ImageMagick/files/4833834/Test.zip)

**Comment by snibgo** [comment]
Please also paste the complete text output from "magick -version".

**Reply by Govind-EIX** [reply]
Unsupported file format or not RAW file `D:\New_folder\DNGTEST_SAMPLE.dng' @ error/dng.c/ReadDNGImage/305.

**Reply by snibgo** [reply]
That isn't the text output from "magick -version".

**Reply by Govind-EIX** [reply]
C:\>magick -version
Version: ImageMagick 7.0.8-3 Q16 x64 2018-06-24 http://www.imagemagick.org
Copyright: Copyright (C) 1999-2018 ImageMagick Studio LLC
License: http://www.imagemagick.org/script/license.php
Visual C++: 180040629
Features: Cipher DPC HDRI Modules OpenMP
Delegates (built-in): bzlib cairo flif freetype gslib heic jng jp2 jpeg lcms lqr lzma openexr pangocairo png ps raw rsvg tiff webp xml zlib

**Comment by snibgo** [comment]
Anyhow, IM v7.0.8-64 with built-in delegate "raw" can't read that DNG. Possibly because the main image has three channels and is JPEG compressed. In other words, it isn't a raw camera image, which is what most raw processors expect.

dcraw can read the file, making a 5184x3456 pixel image, with just 8 bits/channel/pixel. I suggest you use dcraw to read the file. But saving a camera image as a JPEG makes little sense to me.
